packages/ui/lib/components/Form/FieldLayout.tsx

```diff
@@ -60,6 +64,7 @@ export function FieldLayout(props: Props) {
     if (description) {
       return (
         <div id={label} className={descriptionClass}>
+          {optional && <span className="italic mr-1">(optional)</span>}
```

Comment:
i think we can do something like this and just use `required` 
```suggestion
          {!required && <span className="italic mr-1">(optional)</span>}
```

Reply:
actually, we not needed to do it for every input, we can just keep required from -> inputProps.required,

also we need to add `required` also in the `input` field if we are not getting it from `inputProps `